homeassistant/components/mystrom/manifest.json

```diff
@@ -2,9 +2,10 @@
   "domain": "mystrom",
   "name": "myStrom",
   "codeowners": ["@fabaff"],
+  "config_flow": true,
   "dependencies": ["http"],
   "documentation": "https://www.home-assistant.io/integrations/mystrom",
   "iot_class": "local_polling",
   "loggers": ["pymystrom"],
-  "requirements": ["python-mystrom==1.1.2"]
+  "requirements": ["python-mystrom==2.2.0"]
```

Comment:
The library bump should be a preliminary PR.

Reply:
I created the PR #93389 #93389